### PR TITLE
shellz: init at 1.6.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
shellz is a small utility to manage your ssh, telnet, kubernetes, winrm, web or any custom shell in a single place.

https://github.com/evilsocket/shellz

Related to #81418

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 139762` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>shellz</li>
  </ul>
</details>

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 139762` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>shellz</li>
  </ul>
</details>

File: pkgs/tools/security/shellz/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "evilsocket";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1mhl1y0jkycyl1hgwxavxkm1f6kdx1sz3bvpmkr46sdijji06imi";
+  };
+
+  vendorSha256 = "14rd9xd7s5sfmxgv5p9ka8x12xcimv5hrq7hzy0d1c3ddf50rr7n";
```
Comment (figsoda) [inline pkgs/tools/security/shellz/default.nix:17]: included in upstream makefile, see https://github.com/evilsocket/shellz/blob/v1.6.0/Makefile#L6
```suggestion
  vendorSha256 = "14rd9xd7s5sfmxgv5p9ka8x12xcimv5hrq7hzy0d1c3ddf50rr7n";
  
  ldflags = [ "-s" "-w" ];
```

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/tools/security/shellz/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/security/shellz/default.nix:17]: Would only be a 4 MB win (approx. 9 %) for us if we strip the debugging information. As far as I remember is the DWARF symbol table omitted by default in Nix or at least the output looks like that nowadays.

```bash
...:~/shellz]$ du -h shellz-*
35M	shellz-s-w
39M	shellz-w
48M	shellz-unstripped
```

**Pull request merged**